WalletWasabi.Fluent/Helpers/LabelHelpers.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using WalletWasabi.Fluent.ViewModels.Wallets.Labels;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.Helpers;
+
+public static class LabelHelpers
+{
+	public static IEnumerable<(string Label, int Score)> GetMostUsedLabels(this Wallet wallet, Intent intent)
```

Comment:
As a pointer for future improvements: 
This method is pure and encapsulates an algorithm, that is usually implemented following the **Strategy Pattern*. Currently, it's untestable because it uses `Wallet`. It would be necessary to make it independent of `Wallet`, using data that is easier to supply.

Just as a pointer for future improvements: W